src/Explorer/Tree/ResourceTreeAdapter.tsx

```diff
@@ -716,6 +716,19 @@ export class ResourceTreeAdapter implements ReactAdapter {
       },
     ];
 
+    if (item.type === NotebookContentItemType.Notebook) {
+      items.push({
+        label: "Publish to gallery",
+        iconSrc: undefined, // TODO
```

Comment:
>undefined [](start = 17, length = 9)

use same icon as used in publish in command bar?

further, is that icon the correct one?